### PR TITLE
OSDOCS-4926: Doc node port issue workaround procedure

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-4926

Link to docs preview:
https://55215--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-troubleshooting.html#microshift-nodeport-unreachable-workaround_microshift-known-issues

QE review:
Developer preview, external QE not required
SME ack required

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@pacevedom or @jogeo PTAL and let me know if lgtm or needs edits.

**Comment by ShaunaDiaz** [comment]
12fa44e also added a small fix not correctly applied to this version of the docs

File: modules/microshift-nodeport-unreachable-workaround.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+:_content-type: PROCEDURE
+[id="microshift-nodeport-unreachable-workaround_{context}"]
+= Manually restarting the `ovnkube-master` pod to resume node port traffic
+
+There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To troubleshoot this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* Access to the cluster as a user with the `cluster-admin` role.
```
Comment (jogeo) [inline modules/microshift-nodeport-unreachable-workaround.adoc:13]: MicroShift doesn't support users, other than the built in admin. So, there is nothing for the user to actually do here, other than be able to access the cluster using the default kubeconfig file (i.e. set KUBECONFIG), that's generated by MicroShift and available on the host at /var/lib/microshift/resources/kubeadmin/kubeconfig

**Review by jogeo — COMMENTED** [review]
One last comment, the rest of the updates look good; thanks!

File: modules/microshift-nodeport-unreachable-workaround.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-known-issues.adoc
+:_content-type: PROCEDURE
+[id="microshift-nodeport-unreachable-workaround_{context}"]
+= Manually restarting the `ovnkube-master` pod to resume node port traffic
+
+There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To troubleshoot this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```
Comment (EricPonvelle) [inline modules/microshift-nodeport-unreachable-workaround.adoc:8]: ISG recommends against starting a clause with "There is" so maybe:

s/There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped./New installs of {product-title} might experience stopped `NodePort` service traffic.

File: modules/microshift-nodeport-unreachable-workaround.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
```
Comment (EricPonvelle) [inline modules/microshift-nodeport-unreachable-workaround.adoc:13]: You should define the first use of OVN in this topic

File: modules/microshift-nodeport-unreachable-workaround.adoc [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* Access to the `kubeconfig` file.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+Run the commands listed in each step that follows to restore the `NodePort` service traffic after a new {product-title} install:
```
Comment (EricPonvelle) [inline modules/microshift-nodeport-unreachable-workaround.adoc:19]: This and the previous instance of `new` can probably be removed for a different phrasing. Maybe something like

"... restore the `NodePort` service traffic after you complete a {product-title} install"

File: modules/microshift-nodeport-unreachable-workaround.adoc [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+. Find the name of the ovn-master pod that needs to be restarted by running the following command:
```
Comment (EricPonvelle) [inline modules/microshift-nodeport-unreachable-workaround.adoc:21]: You could change "needs to be restarted" to active voice with "that you want to restart"

File: modules/microshift-nodeport-unreachable-workaround.adoc [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+----
+
+. Force a restart of the of the ovnkube-master pod by running the following command:
++
+[source, terminal]
+----
+$ oc -n openshift-ovn-kubernetes delete pod $pod
+----
+
+. Optional: To confirm that a new ovnkube-master pod has been restarted, run the following command:
```
Comment (EricPonvelle) [inline modules/microshift-nodeport-unreachable-workaround.adoc:35]: See previous comments on "new" and passive voice

**Review by EricPonvelle — COMMENTED** [review]
I had some small items to address, but it looks good otherwise.

File: modules/microshift-nodeport-unreachable-workaround.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply modules/microshift-nodeport-unreachable-workaround.adoc:8]: Saying the software might experience something is anthropomorphism (a different style rule). Will consider a change.

**Pull request merged**